samples/gke/slack-command/slack_command.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: extensions.sk8s.io/v1
+kind: Topic
+metadata:
+  name: slack
+---
+
+apiVersion: extensions.sk8s.io/v1
+kind: Function
+metadata:
+  name: slack-command
+spec:
+  image: sk8s/slack_command:test
```

Comment:
would prefer to have the image as `slack-command` since that is also the name of the function created, it also matches what we do for all other sk8s images, maybe also rename this file to `slack-command.yaml`

Reply:
👍